### ISSUE TITLE
Vertical Panes extremely difficult to resize

### ISSUE THREAD

**Issue by jtsom** [rangav/thunder-client-support#305]
**Describe the bug**
When trying to resize the vertical panes (request/response panes), the click point for the resize handle is about 1 pixel, and on a high resolution monitor, it's a game of hit and miss to get the EXACT spot to resize the panes.

**To Reproduce**

Try and click between the vertical panes to resize.

**Expected behavior**
Not having to find the one exact pixel area to grab to resize the panes.

**Platform:** 
 - OS: Windows/MacOS
 - vscode version: Latest Insider 
```
Version: 1.60.0-insider (user setup)
Commit: 5f19eee5dc9588ca96192f89587b5878b7d7180d
Date: 2021-08-24T19:41:58.784Z
Electron: 13.1.8
Chrome: 91.0.4472.164
Node.js: 14.16.0
V8: 9.1.269.39-electron.0
OS: Windows_NT x64 10.0.19043
```
 - 
 - extension version: 1.9.1

**Solution:**
Make the click target larger

**Comment by GreatScott000** [comment]
I am finding the same issue.  Even better would be if it could save the size of the splits when changing request.

**Comment by need47** [comment]
Is it possible to remember the pane sizes/positions? I want a larger response pane. Whenever I start a new request, the request/response panes are restored to 50%.

**Comment by rangav** [comment]
This is now fixed in v1.10.0
https://github.com/rangav/thunder-client-support/releases/tag/v1.10.0

please let me know if any feedback
